kubernetes/redpanda/templates/headless-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
```

Comment:
We might want to split this into two services, one for `kafka`, and one for `rpc`. I suspect they'll have different restrictions at some point. Either that or we don't need rpc here, I changed the `rpk config set` section to not lookup seeds from here.

Reply:
But I think a StatefulSet can only have one `serviceName`, so it can match only one headless service?

Also I tried taking rpc out, but it seems to break things.

Reply:
I'm not sure there's any relation between `statefulSet.spec.serviceName` and the headless service. That matching is done with labels and selector as a convenience for tools that need a broker to connect to.  Reading into it, `serviceName` is used to build a cluster for dns by pod name.  The pod names are predictable, so, maybe, we don't need the other services at all, we can use `redpanda-0.redpanda`, `redpanda-1.redpanda`, etc as the advertised address for rpc as seeds in the configMap.

Reply:
Fixed in: https://github.com/oranda/deployment-automation/pull/3